documentation/assemblies/upgrading/assembly-upgrade-zookeeper.adoc

```diff
@@ -8,6 +8,9 @@
 [role="_abstract"]
 If you are using a ZooKeeper-based Kafka cluster, an upgrade requires an update to the Kafka version and the inter-broker protocol version.
 
+If you want to switch from using a ZooKeeper-based Kafka cluster to a KRaft-based cluster, the steps must be performed separately from the upgrade. 
```

Comment:
This sentence let me think we are suggesting to switch from a cluster to another. Something like "you can move from using a ZooKeeper-based cluster to a different one which is KRaft-based". I think it's confusing. It should be more referring to the migration operation, making clear you are going to use the same cluster but through migration you are moving out from using ZooKeeper for metadata and using KRaft instead.

Reply:
Understood. Changed as follows:

_If you want to switch a kafka cluster from using a ZooKeeper for metadata management to operating in KRaft mode, the steps must be performed separately from the upgrade. 
For information on migrating to a KRaft-based cluster, see xref:proc-deploy-migrate-kraft-str[]._

Reply:
This sounds great!